schema-loader/src/main/java/com/scalar/db/schemaloader/command/CassandraCommand.java

```diff
@@ -55,6 +56,9 @@ public class CassandraCommand extends StorageSpecificCommand implements Callable
       description = "Cassandra replication factor")
   private String replicationFactor;
 
+  @ArgGroup(exclusive = true)
+  DeleteOrRepairTables deleteOrRepairTables;
```

Comment:
We can make it `private`?
```suggestion
  private DeleteOrRepairTables deleteOrRepairTables;
```

The same is also true for the other `XXXCommand`?

Reply:
Yes, thank you for pointing this out. It is revised in https://github.com/scalar-labs/scalardb/pull/622/commits/b6b479940291a124586722c1fd9be51074d422c9